### DISCUSSION TITLE
How can be checked in an CI/CD process whether Pimcore 11.3 is already installed or not?

### DISCUSSION THREAD

**Discussion by code-status-404** [pimcore/pimcore#17605] [General]
Due to the omission of the installation parameter --ignore-existing-config in the Pimcore installer or the system.yaml in the var/config directory, it is no longer possible to easily check whether Pimcore is already installed or not. Therefore, the installer can be run multiple times, which in the worst case can lead to data loss. 

How can now easily checked with Pimcore 11.3 whether it has already been installed or not? Or what could be a best practice to prevent the installer from being run multiple times?

**Comment by dpfaffenbauer** [comment]
why do you need that?

**Reply by code-status-404** [reply]
We install and update Pimcore for our customers via a pipeline. Previously, the procedure was that if no system.yaml was available, the installer was executed (1st installation) and then during the 2nd deployment the installer was executed, but it was aborted because the system.yaml was already present.

**Reply by dpfaffenbauer** [reply]
we do that via bash like:

```bash
bin/console doctrine:query:sql "SELECT * FROM classes"

if [ $? -eq 0 ]; then
    echo "Pimcore is already installed. Just running migration.";
else
    echo "Install Pimcore";
fi
```

**Comment by bluvulture** [comment]
hey @bk-hitcom 

You can add a file upon successful installation ie `touch var/.pimcore.installed` 
Then just check for the existence of it.

**Reply by code-status-404** [reply]
Thanks, that seems to be the easiest solution. We will integrate this into our pipeline and check whether the file exists or not.
